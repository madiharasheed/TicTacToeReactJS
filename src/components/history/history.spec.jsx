import React from "react";
import { shallow } from "enzyme";

import History from "./history";
// Describing History Components
describe("History component", () => {
  it("should render History component", () => {
    const mockCallBack = jest.fn();
    // const history = [{ tiles: Array(9) }];
    const history = [{ tiles: Array(9).fill(null) }];
    // const historyWrap = shallow
    const historyWrapper = shallow(
      <History history={history} onClick={mockCallBack} />
    );
   
  });
});
