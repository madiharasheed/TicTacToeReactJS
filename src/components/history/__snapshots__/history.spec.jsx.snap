// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component should render History component 1`] = `
<div>
  <h1>
    History
  </h1>
  <ol
    className="list-group"
  >
    <li
      className="list-group-item"
      key="0"
    >
      <button
        onClick={[Function]}
      >
        Restart Game
      </button>
    </li>
  </ol>
</div>
`;
